data/coalition outfits.txt

```diff
@@ -129,7 +129,7 @@ outfit "Large Repair Module"
 	"hull repair rate" .57
 	"hull energy" .57
 	"hull heat" .5
-	description "This hub houses a small fleet of repair robots that can climb around on a ship's hull and repair damage to it, even in the middle of combat."
+	description "This hub houses a small fleet of repair robots that can climb around on a ship's hull and repair damage to it, even in the middle of combat…if there is any combat."
```

Comment:
A single character shouldn't be used for ellipses. I also don't find this to be an improvement.